src/listr/tasks/startNodeTaskFactory.js

```diff
@@ -26,18 +26,18 @@ function startNodeTaskFactory(dockerCompose) {
   function startNodeTask(
     config,
     {
-      isFullNode,
       driveImageBuildPath = undefined,
       dapiImageBuildPath = undefined,
       isUpdate = undefined,
-      isMinerEnabled = undefined,
     },
   ) {
     // Check external IP is set
     config.get('externalIp', true);
+    
+    const isMinerEnabled = config.get('core.miner.enable');
 
     if (isMinerEnabled === true && config.get('network') !== NETWORKS.LOCAL) {
-      this.error(`'core.miner.enabled' option supposed to work only with local network. Your network is ${config.get('network')}`, { exit: true });
```

Comment:
I changed it because it wasn't working, did you test it? Here is my output on testnet with miner enabled:

```
# with this.error()
strophy@X250:~/Code/mn-bootstrap$ mn start
✖ this.error is not a function 

# with throw new Error()
strophy@X250:~/Code/mn-bootstrap$ mn start
✖ 'core.miner.enabled' option only works with local network. Your network is testnet. 
```

How do I find out what the current context is? It's impossible to `console.log(this)` because of listr overwriting any console output, and the debugger shows `Error` but not `error` as a function of `this`, so I still don't know what `this` actually refers to.

Reply:
Oh... my bad. This is the context of listr task too since it's in `startNodeTask` what we pass to Listr as a task :).

You can understand the context usually just by reading the code. If it's not clear you can run the command with debugger. In WebStorm it's easy. Just add a Node.JS run configuration with `bin/mn` as a start file and pass `reset` as application params.